### PR TITLE
add gif tab

### PR DESCRIPTION
**Pull request by Rajesh-Sundaram**
# Description
Add Animated Gif's to Demo sample app.

## Links
https://jira.rakuten-it.com/jira/browse/MINIBUILD-740

# Checklist
- [X] I wrote/updated tests for new/changed code.
- [X] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [X] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

File: js-miniapp-sample/src/pages/gifs.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+      iconSrc: require('../assets/images/webp/road.webp'),
+      altLabel: 'infinite_road_webp',
+    },
+  ];
+  return (
+    <Grid
+      container
+      direction="column"
+      justify="flex-start"
+      alignItems="flex-start"
```
Comment (minh-rakuten) [inline js-miniapp-sample/src/pages/gifs.js:42]: I don't know if we can make it become center in mobile webview. I tried with `center`, it's okay for web but in android webview it does not change.

File: js-miniapp-sample/src/pages/gifs.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+      className={classes.grid}
+    >
+      {images.map((it, i) => (
+        <Grid item key={i}>
```
Comment (minh-rakuten) [inline js-miniapp-sample/src/pages/gifs.js:46]: It would be nice if there is gap space between those items.

**Review by minh-rakuten — COMMENTED** [review]
It looks good.

We can improve it more if we have time, for now this page shows all gifs. In my opinion, you can do more work on UI to show the specific gif which users specify.

File: js-miniapp-sample/src/pages/gifs.js (hunk 2 above)
Comment (mohanadwalo) [reply js-miniapp-sample/src/pages/gifs.js]: @minh-rakuten fixed

File: js-miniapp-sample/src/pages/gifs.js (hunk 1 above)
Comment (mohanadwalo) [reply js-miniapp-sample/src/pages/gifs.js]: @minh-rakuten fixed

**Review by minh-rakuten — APPROVED** [review]
LGTM.

**Review by corycaywood — APPROVED** [review]
LGTM 👍

**Pull request merged**